docs/contributing-to-airbyte/building-new-connector/testing-connectors.md

```diff
@@ -26,18 +26,44 @@ See all the test cases and their description in [Standard Source Tests](standard
 
 ### Setting up standard tests for your connector
```

Comment:
can we separate the doc changes with the full documentation PR?

Reply:
ok